README.md

```diff
@@ -80,5 +80,45 @@ After exporting above parameters execute the following commands
 
   make
 
-
+## Build the hive module for QAT 
```

Comment:
Let's reconstruct it a little bit. 
Please introduce a new level. ## Build for CDP DC 7,0
And move the rest part under this new section with lower level.